packages/client-config/src/client-config-types/client_config.ts

```diff
@@ -29,39 +29,39 @@ export type ClientConfigLegacy = Partial<
 /**
  * Versions of client config schema supported by "this" package version.
  * Create union for supported versions here, such as
- * ClientConfig = clientConfigTypesV1.AWSAmplifyGen2BackendOutputs | clientConfigTypesV2.AWSAmplifyGen2BackendOutputs;
+ * ClientConfig = clientConfigTypesV1.AWSAmplifyBackendOutputs | clientConfigTypesV2.AWSAmplifyBackendOutputs;
  */
 export type ClientConfig = clientConfigTypesV1.AWSAmplifyBackendOutputs;
 
 export { clientConfigTypesV1 };
 
-export enum ClientConfigVersions {
+export enum ClientConfigVersionOption {
   LEGACY = '0',
```

Comment:
```suggestion
  V0 = '0',
```

Reply:
Just to give a little more explanation for my preference here:
I think exposing the "legacy" identifier in this type is leaky. There are 2 formats and the fact that 0 is a "special case legacy format" is more of an implementation detail of how we are generating the config, it doesn't need to be elevated to the config version definition itself.

Reply:
Updated.